fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/configs/tokenizer_config.yaml

```diff
@@ -79,15 +79,15 @@ data:
     # AA_tokenizer_path:      "/dccstor/fmm/users/vadimra/dev/code/fuse-drug/fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/tokenizer/pretrained/simple_protein_tokenizer_wrapped_special_AA.json"
     # SMILES_tokenizer_path:  "/dccstor/fmm/users/vadimra/dev/code/fuse-drug/fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/tokenizer/pretrained/bpe_tokenizer_trained_on_chembl_zinc_with_aug_4272372_samples_balanced_1_1.json"
     primary_tokenizer: AA #which tokenizer from the ones below to use as the prime one, i.e. the one whose indices are left unchanged
-    modular_tokenizers_out_path: "/dccstor/fmm/users/vadimra/dev/code/fuse-drug/fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/tokenizer/pretrained/modular_AA_SMILES/"
+    modular_tokenizers_out_path: "/dccstor/fmm/users/vadimra/dev/code/fuse-drug/fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/tokenizer/pretrained/modular_wordlevelAA_BPESMILES/"
```

Comment:
same here; we can't specify the full path in published example.

Reply:
cleaned

Reply:
You have more - search for dccstor